assets/src/containers/ResourcesAccessed.js

```diff
@@ -81,6 +84,16 @@ function ResourcesAccessed (props) {
     [saveSettingClicked]
   )
 
+  const onChangeResourceTypeHandler = event => {
+    setDataLoaded(false)
+    const value = event.target.value
+    if (event.target.checked && !resourceTypeFilter.includes(value)) {
```

Comment:
Could you extract `event.target.checked` into a variable, similar to how `event.target.value` is extracted above?

Reply:
Done